docs/schema/DiscreteControlCondition.schema.json

```diff
@@ -27,10 +27,10 @@
             "format": "default",
             "anyOf": [
                 {
-                    "type": "number"
+                    "type": "null"
```

Comment:
Are these schema changes expected?

Reply:
This is strange, it seems my laptop likes a different order than @evetion and yours: https://github.com/Deltares/Ribasim/pull/574/commits/54273a0c2176ed461c2bb8515d561262e5c7bf65

Reply:
Mmmh, this is annoying.
Let's hope that it was just some version mismatch and moving to pixi solves that.